src/test/mediapipeflow_test.cpp

```diff
@@ -95,6 +95,66 @@ class MediapipeFlowDummyTest : public MediapipeFlowTest {
     }
 };
 
+class MediapipeFlowDummySeparateConfigTest : public MediapipeFlowTest {
+public:
+    void SetUp() {
+        SetUpServer("/ovms/src/test/mediapipe/config_mediapipe_dummy_adapter_full_separate_config.json");
+    }
+};
+
+TEST_P(MediapipeFlowDummySeparateConfigTest, Infer) {
+    const ovms::Module* grpcModule = server.getModule(ovms::GRPC_SERVER_MODULE_NAME);
+    KFSInferenceServiceImpl& impl = dynamic_cast<const ovms::GRPCServerModule*>(grpcModule)->getKFSGrpcImpl();
+    ::KFSRequest request;
+    ::KFSResponse response;
+
+    const std::string modelName = GetParam();
+    request.Clear();
+    response.Clear();
+    inputs_info_t inputsMeta{{"in", {DUMMY_MODEL_SHAPE, precision}}};
+    preparePredictRequest(request, inputsMeta);
+    request.mutable_model_name()->assign(modelName);
+    ASSERT_EQ(impl.ModelInfer(nullptr, &request, &response).error_code(), grpc::StatusCode::OK);
+    auto outputs = response.outputs();
+    ASSERT_EQ(outputs.size(), 1);
+    ASSERT_EQ(outputs[0].name(), "out");
+    ASSERT_EQ(outputs[0].shape().size(), 2);
+    ASSERT_EQ(outputs[0].shape()[0], 1);
+    ASSERT_EQ(outputs[0].shape()[1], 10);
+    std::vector<float> requestData{0., 0., 0, 0., 0., 0., 0., 0, 0., 0.};
+    checkDummyResponse("out", requestData, request, response, 1, 1, modelName);
+}
+
+class MediapipeFlowDummyDummyInSubconfigAndConfigTest : public MediapipeFlowTest {
+public:
+    void SetUp() {
+        SetUpServer("/ovms/src/test/mediapipe/config_mediapipe_dummy_adapter_full_dummy_in_both_config_and_subconfig.json");
+    }
+};
+
+TEST_P(MediapipeFlowDummyDummyInSubconfigAndConfigTest, Infer) {
```

Comment:
What happens in this scenario? We could have in subconfig model dummy with shape (1,12), and in main config shape: (1,10). Which model will be loaded and if there will be warning/information in logs that would ease debug?

Reply:
I think that the subconfig model will be loaded. For now we can leave it as known issue since this is preview but ideally we should report issue that you cannot have the same model configured in main json & subconfig.
For now add test documenting the behaviour whichever it is (expecting shape from subconfig or main config).